pyanaconda/modules/payload/live/live_os.py

```diff
@@ -75,6 +86,17 @@ def set_image_path(self, image_path):
         self.image_path_changed.emit()
         log.debug("LiveOS image path is set to '%s'", self._image_path)
 
+    @property
+    def space_required(self):
+        """Get space required for the source image.
+
+        TODO: Add missing check if source is ready. Until then you shouldn't call this when
+        source is not ready.
+
+        :rtype: int
+        """
+        return getDirSize("/") * 1024
```

Comment:
The function `getDirSize` seems to be used only by this payload. I suggest to move it to the module and rename it to `get_dir_size`.

Reply:
Fixed.